packages/build-tools/src/utils/project.ts

```diff
@@ -42,6 +42,20 @@ export async function setup<TJob extends Job>(ctx: BuildContext<TJob>): Promise<
     ctx.logger.info('Using app configuration:');
     ctx.logger.info(JSON.stringify(appConfig, null, 2));
   });
+
+  await ctx.runBuildPhase(BuildPhase.RUN_EXPO_DOCTOR, async () => {
+    const spawnOptions = {
+      cwd: ctx.reactNativeProjectDirectory,
+      logger: ctx.logger,
+      env: ctx.env,
+    };
+    ctx.logger.info('Running "expo doctor"');
+    try {
+      await ctx.runExpoCliCommand('doctor', spawnOptions);
+    } catch (err) {
+      ctx.logger.error({ err }, 'Command "expo doctor" failed.');
```

Comment:
I assume the command prints to stderr in case of error?

Reply:
yes, this exception should never throw, I added it just in case